geometry_msgs/Vector3 PositionDestination
geometry_msgs/Vector3 AngleDestination
float64 time
float64 points
